server/src/internalClusterTest/java/org/opensearch/snapshots/CloneSnapshotIT.java

```diff
@@ -560,14 +560,14 @@ public void testStartCloneWithSuccessfulShardClonePendingFinalization() throws E
         blockMasterOnWriteIndexFile(repoName);
```

Comment:
Change method name `blockMasterOnWriteIndexFile` to `blockClusterManagerOnWriteIndexFile`?

Reply:
It's a public method, and it has been published to maven, so I plan to change the name in next major version (3.0), and it's tracked in the issue #1684